tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/FutureChecker.java

```diff
@@ -79,7 +78,7 @@ public FutureChecker(int sleepMillis) {
     public final class TestFuture implements Future<String> {
 
         private AtomicBoolean done = new AtomicBoolean(false);
-        private String result = "";
+        private String result = "RESULT";
```

Comment:
This could be `static final` as it's now a constant.

Reply:
Good point, will fix!

Reply:
I just inlined the constant into the two `get` methods.